### ISSUE TITLE
incorrect warnings in array destructuring assignment

### ISSUE THREAD

**Issue by enotsimon** [sirbrillig/phpcs-variable-analysis#48]
this code
```[$a, $b] = [1, 2];```
cause fail warnings
```
Variable $a is undefined.
Variable $b is undefined.
```
version is: 2.1.2

**Comment by sirbrillig** [comment]
Thanks for the report!
